### PR TITLE
feat(elasticbeanstalk): add new check `elasticbeanstalk_managed_platform_updates_enabled`

### PR DESCRIPTION
**Pull request by MarioRgzLpz**
### Context

Added a new check to verify whether managed platform updates are enabled for an Elastic Beanstalk environment. The check fails if no managed platform updates are enabled. The check passes if any type of platform update is enabled. 

Enabling managed platform updates ensures that the latest available platform fixes, updates, and features for the environment are installed. Keeping up to date with patch installation is an important step in securing systems.

### Description

Added new check `elasticbeanstalk_managed_platform_updates_enabled` with respective unit tests and metadata.

### Checklist

- Are there new checks included in this PR? Yes
    - If so, do we need to update permissions for the provider? No.
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [x] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by MrCloudSec — DISMISSED** [review]

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.metadata.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "elasticbeanstalk_managed_platform_updates_enabled",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.metadata.json]:
```suggestion
  "CheckID": "elasticbeanstalk_environment_managed_updates_enabled",
```

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            report.region = environment.region
+            report.resource_id = environment.name
+            report.resource_arn = environment.arn
+            report.resource_tags = environment.tags
+            report.status = "PASS"
+            report.status_extended = f"Elastic Beanstalk environment {environment.name} has automated managed platform updates enabled."
+
+            if environment.managed_platform_updates != "true":
+                report.status = "FAIL"
+                report.status_extended = f"Elastic Beanstalk environment {environment.name} does not have automated managed platform updates enabled."
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py]:
```suggestion
                report.status_extended = f"Elastic Beanstalk environment {environment.name} does not have managed platform updates enabled."
```

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    def execute(self):
+        findings = []
+        for environment in elasticbeanstalk_client.environments.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = environment.region
+            report.resource_id = environment.name
+            report.resource_arn = environment.arn
+            report.resource_tags = environment.tags
+            report.status = "PASS"
+            report.status_extended = f"Elastic Beanstalk environment {environment.name} has automated managed platform updates enabled."
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py]:
```suggestion
            report.status_extended = f"Elastic Beanstalk environment {environment.name} has managed platform updates enabled ."
```

Comment (MarioRgzLpz) [reply prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py]: [3ed20e8](https://github.com/prowler-cloud/prowler/pull/5324/commits/3ed20e8f2de6a01667e4e93520b90dd02395b166)

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py (hunk 2 above)
Comment (MarioRgzLpz) [reply prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.py]: [3ed20e8](https://github.com/prowler-cloud/prowler/pull/5324/commits/3ed20e8f2de6a01667e4e93520b90dd02395b166)

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.metadata.json (hunk 1 above)
Comment (MarioRgzLpz) [reply prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_managed_platform_updates_enabled/elasticbeanstalk_managed_platform_updates_enabled.metadata.json]: [3ed20e8](https://github.com/prowler-cloud/prowler/pull/5324/commits/3ed20e8f2de6a01667e4e93520b90dd02395b166)

**Comment by MarioRgzLpz** [comment]
@sergargar Requested changes done [3ed20e8](https://github.com/prowler-cloud/prowler/pull/5324/commits/3ed20e8f2de6a01667e4e93520b90dd02395b166)

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**